### PR TITLE
Create header

### PR DESCRIPTION
**Pull request by Aidan-bn**
# Portfolio: Setup and Mobile Version Skeleton 

This is a simple web development project for Microverse created using simple HTML (Hyper Text Mark-up Language) and CSS (Cascading Style Sheet)

## Built With

Major languages: HTML and CSS

[Live Demo Link] (https://livedemo.com)

## Getting Started
To get a local copy up and running follow these simple example steps.

👤 **Author1**

- GitHub: [@aidan-bn](https://github.com/aidan-bn)
- Twitter: [@aidan-bn](https://twitter.com/aidan-bn)
- LinkedIn: [Aidan-bn](https://linkedin.com/in/aidan-bn)

### CODE REVIEW

**Review by t-yanick — COMMENTED** [review]
Hi @Aidan-bn, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have a professional readme file. Checkout this [template](https://github.com/microverseinc/readme-template) and adapt it to your project. Replace all the information in the various sections of the template with information pertaining to your poject.  This will make your readme file more informative and professional. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    <link rel="stylesheet" href="/styles.css" >
+    <title>Portifolio</title>
+</head>
+<body>
+    <header class="nav">
+        <div class="navbar">
+            <ul class="list">
+                <li class="listitems">Portfolio</li>
+                <li class="listitems">About</li>
+               <li class="listitems">Contact</li>
```
Comment (micheaol) [inline index.html:20]: There should be a hamburger button on the right and has a `nav` tag like the code below:

````
<nav>
     hamburger button
</nav>
````

Your Implementation | Figma Design
:---:|:---:
image | image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+            </ul>
+        </div>
+
+        <section class="headline">
```
Comment (micheaol) [inline index.html:24]: Please ensure to remove the `headline section` from the header. Please see the code below:

````
<header>
       <nav>
  ------------------------------
        </nav>
</header>

<section>
        ------------------headlines----------------
</section>








````

File: index.html [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+            <div class="primary">
+                <div class="primaryblack">
+                    <h1>Hey there. I'm Aidan</h1>
+                </div>
```
Comment (micheaol) [inline index.html:28]: Please ensure that you stick to the design of any of the templates of your choice as much as possible (e.g. font, colors, images, tests, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Your Implementation | Figma Design
:---:|:---:
image | image

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+  list-style: none;
+  margin-right: 8px;
+}
+
+.headline {
+  width: 70vw;
+  height: 60vh;
+  margin: auto;
+  padding-top: 50px;
+}
```
Comment (micheaol) [inline styles.css:33]: Please ensure that the `headline content` is aligned with flexbox according to the project requirement.

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @Aidan-bn , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+            </div>
+            <div class="hamburger">
+                <div class="navbar">
+                </div>
+                <div class="navbar">
+                </div>
+                <div class="navbar">
+                </div>
+            </div>  
+        </nav>
```
Comment (tufoinnkuo10) [inline index.html:28]: - [ ] like the previous reviewer mentioned please only the hamburger icon should be inside the nav tag and not the logo.
```
<header>
    <p>logo</p>
     <nav>
           hamburger button
   </nav>
</header>
```

File: index.html [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+                    <p class="para"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                        don’t hestiate to contact me.</p>
+                </div>
+                <div class="socialmedia">
+                    <ul class="ul-socialmedia">
+                        <li class="socialists"><a href="https://twitter.com/aidan-bn"><i class="fa fa-twitter"></i></a></li>
+                        <li class="socialists"><a href="https://twitter.com/aidan-bn"><i class="fa fa-linkedin"></i></a></li>
+                        <li class="socialists"><a href="https://maxcdn.com/aidan-bn"><i class="fa fa-maxcdn"></i></a></li>
+                        <li class="socialists"><a href="https://github.com/aidan-bn"><i class="fa fa-github"></i></a></li>
+                        <li class="socialists"><a href="https://angellist.com/aidan-bn"><i class="fa fa-angellist"></i></a></li>
```
Comment (tufoinnkuo10) [inline index.html:50]: - [ ] please like the previous reviewer mentioned you should follow same figma font of margin, colors, font weight and sizes. also your social icons are different from that of the figma design. 
your design
Screenshot (95)_LI
figma design
Screenshot (96)_LI

**Review by KayLemba — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hello  @Aidan-bn     , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

HIGHLIGHTS:

- Your project is professional 
- Your repo is professional 
- Well structured files 


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

[OPTIONAL] Kindly add a screenshot to your README file showing your webpage user Interface. You can use extentions like fireshot to capturer the entire page.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**